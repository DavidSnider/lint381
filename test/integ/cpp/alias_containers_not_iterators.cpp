using Foo_t = std::vector<int>::iterator;

using Bar_t = std::vector<int>;
