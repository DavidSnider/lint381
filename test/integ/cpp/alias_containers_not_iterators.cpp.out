alias_containers_not_iterators.cpp:1:31: error: Create type aliases for containers, not iterators
using Foo_t = std::vector<int>::iterator;
                              ^^^^^^^^^^
